src/test/java/picocli/CommandLineTest.java

```diff
@@ -3619,6 +3623,61 @@ public void testAtFileExpandedAbsolute() {
         assertEquals(Arrays.asList("1111", "2222", ";3333"), app.files);
     }
 
+    @Test
+    public void testAtFileSimplified() {
+        System.setProperty("picocli.useSimplifiedAtFiles", "true");
+        class App {
+            @Option(names = "--simpleArg")
+            private boolean simple;
+
+            @Option(names = "--argWithSpaces")
+            private String withSpaces;
+
+            @Option(names = "--quotedArg")
+            private String quoted;
+
+            @Option(names = "--multiArg")
+            private List<String> strings;
+
+            @Option(names = "--urlArg")
+            private URL url;
+
+            @Option(names = "--unescapedBackslashArg")
+            private String unescaped;
+        }
+        File file = findFile("/argfile-simplified.txt");
+        App app = CommandLine.populateCommand(new App(), "@" + file.getAbsolutePath());
+        assertTrue(app.simple);
+        assertEquals("something with spaces", app.withSpaces);
+        assertEquals("\"something else\"", app.quoted);
+        assertEquals(Arrays.asList("something else", "yet something else"), app.strings);
+        assertEquals("https://picocli.info/", app.url);
+        assertEquals("C:\\Program Files\\picocli.txt", app.unescaped);
+    }
+
+    @Test
+    public void testAtFileSimplifiedWithQuotesTrimmed() {
+        System.setProperty("picocli.useSimplifiedAtFiles", "true");
+        class App {
+            @Option(names = "--quotedArg")
+            private String quoted;
+
+            @Option(names = "--urlArg")
+            private URL url;
+
+            @Option(names = "--unescapedBackslashArg")
+            private String unescaped;
+        }
+        File file = findFile("/argfile-simplified-quoted.txt");
+        final App app = new App();
+        final CommandLine cli = new CommandLine(app);
+        cli.setTrimQuotes(true);
```

Comment:
@remkop Any chance this could also be set through a system property? This way, it prevents Picocli to be used in the simple way: `CommandLine.call(...)`

Reply:
Yes sure. Do you want to provide a separate pull request for that?

Reply:
I'll take a look in another PR.